src/components/Checklist/Checklist.cy.tsx

```diff
@@ -23,22 +23,42 @@ const CHECKBOXES = [
 ];
 
 const CHECKLIST_ID = '[data-test-id=checklist]';
+const CHECKBOX_ID = '[data-test-id=checkbox-icon-box]';
+const CHECKBOX_INPUT_ID = '[data-test-id=checkbox-input]';
 
-const Component = ({ direction, columns }: { direction: ChecklistDirection; columns?: Columns }): ReactElement => {
-    const [activeBoxes, setActiveBoxes] = useState<string[]>([]);
+const Component = ({
+    direction,
+    columns,
+    activeValues,
+    setActiveValues,
+    ariaLabel,
+    'data-test-id': dataTestId,
+}: {
+    ariaLabel?: string;
+    direction?: ChecklistDirection;
+    columns?: Columns;
+    activeValues?: string[];
+    setActiveValues?: () => void;
+    'data-test-id'?: string;
+}): ReactElement => {
+    const [activeBoxes, setActiveBoxes] = useState<string[]>(activeValues ? activeValues : []);
 
     return direction === ChecklistDirection.Horizontal ? (
```

Comment:
The Ternary condition with the ChecklistDirection seems unnecessary.
The component could be simplified like this:
```
const [activeBoxes, setActiveBoxes] = useState<string[]>(activeValues ? activeValues : []);

    return (
        <Checklist
            ariaLabel={ariaLabel}
            data-test-id={dataTestId}
            activeValues={activeBoxes}
            setActiveValues={setActiveValues ?? setActiveBoxes}
            direction={direction ?? ChecklistDirection.Vertical}
            checkboxes={CHECKBOXES}
            columns={columns}
        />
    );
 ```